services/connected_account.yaml

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016-present, CloudZero, Inc. All rights reserved.
+# Licensed under the BSD-style license. See LICENSE file in the project root for full license information.
+
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CloudZero Connected Account Template
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: CloudZero Cross Account Access
+        Parameters:
+          - ExternalId
+          - ReactorAccountId
+      - Label:
+          default: CloudZero Callback Information
+        Parameters:
+          - ReactorSQSQueueUrl
+    ParameterLabels:
+      ExternalId:
+        default: |
+          Unique ExternalId for your Organization; Generated by CloudZero; Used for Cross Account Roles
+      ReactorAccountId:
+        default: |
+          CloudZero AWS AccountID; Used for Cross Account Roles
+      ReactorSQSQueueUrl:
+        default: |
+          CloudZero Reactor SQS Queue receives callbacks from this template
+
+Parameters:
+  ExternalId:
+    Type: String
+    Description: |
+      Unique ExternalId for Customer Organization; for cross-account Role Access and
+      associating this template with a Customer Organization
+  ReactorAccountId:
+    Type: String
+    Default: 061190967865
+    Description: |
+      CloudZero AWS AccountID; for cross-account Role Access
+  ReactorSQSQueueUrl:
+    Type: String
+    Default: Does not Exist Yet
+    Description: |
+      CloudZero Reactor Queue Url; receives callbacks from this template
+
+Resources:
+  Disovery:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      Tags:
+        - Key: cloudzero
+          Value: cloudzero
+      TemplateURL: https://s3.amazonaws.com/cz-provision-account/services/latest/discovery.yaml
+
+  ResourceOwnerAccount:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      Tags:
+        - Key: cloudzero
+          Value: cloudzero
```

Comment:
this k:v pair looks dubious...  probably should say cloudzero *\something*, but what exactly are we trying to communicate?

Reply:
I agree ... I did a quick googling for AWS 3rd Party Tagging best practices but didn't find much. I want to make sure our customers can easily find our resources.

Reply:
Maybe Erik has an opinion.